dev-util/rebar/rebar-3.16.0.ebuild

```diff
@@ -0,0 +1,91 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit bash-completion-r1
+
+HEX_PACKAGES="
+	bbmustache-1.10.0
+	certifi-2.6.1
+	cf-0.3.1
+	cth_readable-1.5.1
+	erlware_commons-1.5.0
+	eunit_formatters-0.5.0
+	getopt-1.0.1
+	meck-0.8.13
+	providers-1.8.1
+	relx-4.4.0
+	ssl_verify_fun-1.1.6
+"
+
+hex_package_uris() {
+	local package
+	for package in "$@"; do
+		echo "https://repo.hex.pm/tarballs/$package.tar -> $package.hex.tar"
+	done
+}
+
+DESCRIPTION="Erlang build tool that makes it easy to build and test applications and releases"
+HOMEPAGE="https://www.rebar3.org/ https://github.com/erlang/rebar3/"
+SRC_URI="https://github.com/erlang/rebar3/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://distfiles.matthew.run/dev-util/rebar/${PV}/packages.idx.xz -> ${P}-packages.idx.xz
+	$(hex_package_uris ${HEX_PACKAGES})"
+S="${WORKDIR}/rebar3-${PV}"
+
+LICENSE="Apache-2.0 MIT BSD"
```

Comment:
(Just making sure: You added licenses for all the bundled hex packages too?)

Reply:
Yep.

Rebar 3 itself is Apache 2, and the deps are:

* bbmustache: MIT
* certifi: BSD
* cf: MIT
* cth_readable: BSD
* erlware_commons: Apache 2, MIT
* eunit_formatters: Apache 2
* getopt: BSD
* meck: Apache 2
* providers: Apache 2
* relx: Apache 2
* ssl_verify_fun: MIT